e2e/pages/resources/cms-lookup/cms-lookup.readonly.spec.js

```diff
@@ -16,7 +16,7 @@ beforeAll(async () => {
 });
 
 describe('On cms reverse look up page', () => {
-    for (var i in inputs) {
+    for (const i in inputs) {
```

Comment:
This looks weird. `i` is usually the indexing variable, which _must_ change. But `const` variables don't change. What's going on here?

Reply:
so beforeall in the describe block seems to be changing i values and running it for only third values from array(I think its running in async mode but not positive) so i have to keep i as const through out one describe block.
I value will still get increased after iteration.

Reply:
What about?
```suggestion
    inputs.forEach(input => {
```

Then, anywhere below you're using `inputs[i]` you can replace it with just `input`?

Reply:
Tried. It sys 0 test and doesnt run anything